integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -79,9 +83,21 @@ def __init__(
         self._persist_path = persist_path
         self._distance_function = distance_function
         # Create the client instance
-        if persist_path is None:
-            self._chroma_client = chromadb.Client()
+        # Ensure only one of persist_path or host is provided
+        if persist_path and host:
+            error_message = "You cannot provide both `persist_path` and `host`. Please choose one."
+            raise ValueError(error_message)
+        if host:
+            # Remote connection via HTTP client
+            self._chroma_client = chromadb.HttpClient(
+                host=host,
+                port=port or 8000,
+            )
```

Comment:
```suggestion
        if host and port is not None:
            # Remote connection via HTTP client
            self._chroma_client = chromadb.HttpClient(
                host=host,
                port=port,
            )
```

Let's be explicit: in case users want to connect to a remote Chroma instance, they should specify both `host` and `port`.